### ISSUE TITLE
Chat bubble text is getting trimmed and shifts to next line.

### ISSUE THREAD

**Issue by MonicaChatbot** [microsoft/BotFramework-WebChat#4464]
Steps to replicate:
1. go to the page:https://www.harvardpilgrim.org/public/home
2. Open the chat box  at bottom right corner( there is red bar click anywhere to open it).
3. Then type "my plan" and click on send .
4. They which we get from bot , the last word is trimmed and trimmed part is displayed in next line.
Below find the screen shot, the trimmed words are highlighted in yellow.

![image](https://user-images.githubusercontent.com/116090495/196454912-a827dc2c-099b-4bb1-982d-16b3abfddbc2.png)

On inspecting the element i could see  "&nbsp" is added after each word, not sure why? Below find the screenshot for reference.Could you help  in debugging the issue?
![image](https://user-images.githubusercontent.com/116090495/196456438-18c09df1-37bf-44c4-85fa-862113cc8ed3.png)

**Comment by stevkan** [comment]
@MonicaChatbot - Unfortunately, I am unable to repro the issue. I can only assume that there is a property being passed into the CSS elsewhere on the page that is somehow affecting how word-break is being applied in the chat bubbles.

There is a Web Chat styleOptions property called `messageActivityWordBreak` that does work for me in that if I set it to 'break-all' then it breaks the words however they fall at the end of each line. But if I set it to 'break-word' then each word is respected.

I browsed thru the available code for the page you linked above looking for the Web Chat code and the CSS and it doesn't appear that you are passing 'word-break' anywhere. Web Chat defaults to setting the above property to 'break-word'. But, again, something elsewhere may be interfering with that...?

Try passing that property in to the styleOptions using 'break-word' and see if that makes a difference for you. If it doesn't, are you able to share your code so that I could take a closer look at it?
